frontend/src/modules/machinery/reducer.js

```diff
@@ -35,7 +41,8 @@ function dedupedTranslations(
         const sameTranslation = translations.findIndex(
             (oldT) =>
                 newT.original === oldT.original &&
-                newT.translation === oldT.translation,
+                newT.translation === oldT.translation &&
+                newT.projectName === oldT.projectName,
```

Comment:
IIUC (and testing locally confirms it), we'll show separate entries for each of the items in the example given at https://github.com/mozilla/pontoon/pull/1794#issuecomment-766412395.